modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -1779,12 +1779,43 @@ The fundamental matrix may be calculated using the cv::findFundamentalMat functi
 */
 CV_EXPORTS_W double sampsonDistance(InputArray pt1, InputArray pt2, InputArray F);
 
-/** @brief Computes an optimal affine transformation between two 3D point sets.
+/** @brief Computes an optimal affine transformation between two 3D point sets:
+\f[
```

Comment:
Please avoid using formulas in "@brief" block (check corruption of the [Functions table](http://pullrequest.opencv.org/buildbot/export/pr/10633/docs/d9/d0c/group__calib3d.html) with overview of these functions).
Add extra empty line after it and/or extra wording like "Details:".

Reply:
Thank you for the review.

Changes have been made according to the comments.